terraform/examples/module_depends_on/main.tf

```diff
@@ -0,0 +1,30 @@
+
+## normal module usage without hidden external dependencies
+
+# will be created/destroyed as soon as possible
+module "has-no-dependency" {
+  source  = "../../modules/terraform-null-resources"
+
+  string = "Whenever!"
+}
+
+# will be created/destroyed as soon as possible
+resource "null_resource" "is-no-dependency" {}
+
+## make use of module_depends_on to define hidden external dependencies
```

Comment:
Let's use the same comment syntax that we also use for our [modules](https://github.com/mineiros-io/terraform-module-template/blob/master/modules/vpc/main.tf#L11)?

Reply:
fixed